kore/src/Kore/Repl.hs

```diff
@@ -278,7 +279,7 @@ runRepl
     catchEverything a =
         Exception.handleAll $ \e -> liftIO $ do
             putStrLn "Stepper evaluation errored."
-            print e
+            print (Pretty.prettyException e)
```

Comment:
I think this is fine, but we should at least try logging the exceptions as is done [here](https://github.com/kframework/kore/blob/a1adb18dd2d67307c4dcdfc02e171b8f924c5eb2/kore/app/exec/Main.hs#L591), but without re-throwing the exception.